Nos/Service/CurrentUser+PublishEvents.swift

```diff
@@ -203,20 +194,61 @@ extension CurrentUser {
         await publishContactList(tags: stillFollowingKeys.pTags)
     }
     
-    @MainActor func publishRequestToVanish(to relays: [URL]? = nil, reason: String? = nil) async throws {
+    @MainActor func publishAccountDeletedMetadata() async throws {
+        guard let author else {
+            Log.error("Error: no author")
+            return
+        }
+        
+        author.about = nil
+        author.displayName = "Account deleted"
+        author.name = "Account deleted"
+        author.website = nil
+        author.nip05 = nil
+        author.profilePhotoURL = nil
+        author.uns = nil
+        author.rawMetadata = nil
+        
+        try viewContext.save()
+        try await publishMetadata()
+    }
+    
+    @MainActor func publishEmptyFollowList() async throws {
+        guard let author else {
+            Log.error("Error: no author")
```

Comment:
Maybe a little more context in the log message would help. Same above -- since both messages are the same, I'm not sure I'd know where to go if I saw this message in the console. Something like "Error: Can't publish empty follow list since author is nil", but I'm sure you can come up with something better than that.